linera-base/Cargo.toml

```diff
@@ -19,6 +19,7 @@ bcs = { workspace = true }
 ed25519-dalek = { workspace = true }
 generic-array = { workspace = true }
 hex = { workspace = true }
+prometheus = { workspace = true }
```

Comment:
move to the section below `[target.'cfg(not(target_arch = "wasm32"))'.dependencies]`

Reply:
I'm having some trouble with this. If I move it down, `prometheus_util` apparently *will* be called from wasm code, because I've tried to make it compile, but it seems like I need to add this `#[cfg(not(target_arch = "wasm32"))]` in a bunch of places... 🤔
Is that the expected here? Is there an easier solution?